MekHQ/src/mekhq/gui/dialog/CustomizeScenarioDialog.java

```diff
@@ -88,6 +88,9 @@ public CustomizeScenarioDialog(JFrame parent, boolean modal, Scenario s, Mission
             scenario = s;
             newScenario = false;
         }
+        if (newScenario) {
+            scenario.setStatus(ScenarioStatus.NEW);
+        }
         campaign = c;
         if (scenario.getDate() == null) {
             scenario.setDate(campaign.getLocalDate());
```

Comment:
This will give status "Pending" to all new scenarios but the current date will still be assigned. Functionality would need to be added to the dialog to allow the user to pick a date. Furthermore, what is the distinction no between "Pending" and "Current?" Is the intent of "Current" that we are on the actual day the scenario should happen? If so, we would also need code that automatically switched from pending to current where appropriate  on newDay. The code would also need to check when executing the dialog to determine if its current or pending.